scheduled-jobs/build/poll-payload/Jenkinsfile

```diff
@@ -91,6 +90,12 @@ node() {
                 description += " [no change]\n"
                 continue
             }
+            build(
```

Comment:
can this be restricted to just the x86_64 releases? we only want to sync rpms for accepted x86_64 releases.